My Spotify/My Spotify/Coordinators/Storyboarded.swift

```diff
@@ -14,6 +14,7 @@ enum Storyboard: String {
     case viewPlaylist = "ViewPlaylist"
     case search = "Search"
     case trackOptions = "TrackOptions"
+    case viewPorfile = "UserArtistProfile"
```

Comment:
Keep variable name same as it's value to avoid ambiguity.
```suggestion
    case userArtistProfile = "UserArtistProfile"
```